pkg/cache/cache.go

```diff
@@ -118,44 +119,59 @@ type Options struct {
 	// Mapper is the RESTMapper to use for mapping GroupVersionKinds to Resources
 	Mapper meta.RESTMapper
 
-	// Resync is the base frequency the informers are resynced.
+	// ResyncEvery is the base frequency the informers are resynced.
 	// Defaults to defaultResyncTime.
-	// A 10 percent jitter will be added to the Resync period between informers
+	// A 10 percent jitter will be added to the ResyncEvery period between informers
 	// So that all informers will not send list requests simultaneously.
-	Resync *time.Duration
+	ResyncEvery *time.Duration
 
-	// Namespace restricts the cache's ListWatch to the desired namespace
-	// Default watches all namespaces
-	Namespace string
+	// View restricts the cache's ListWatch to the desired fields per GVK
+	// Default watches all fields and all namespaces.
+	View ViewOptions
```

Comment:
I am really, really not a fan of moving thins into substructs. It is much easier to find what options exist if they are on the top level.

Reply:
Do you have a better name? Having all of the selectors and transformers at the top level IMO it's super confusing for new users. It's also not clear at all that all Get() and List() requests would be limited to those for example